### DISCUSSION TITLE
TuYa TS0601_switch is not added as known.

### DISCUSSION THREAD

**Discussion by leha123ra** [Koenkk/zigbee2mqtt#10071] [Devices]
Hello!
I bought a device. It looks like [this](https://www.zigbee2mqtt.io/devices/TS0601_switch.html#tuya-ts0601-switch). It's called like this.
But it is not added as known.
I have updated Z2M. Now version 1.22.1-dev commit: b4af832.
I am newbie in zigbee and home automation.
What should I do to make this device work correctly on the network?
I've tried deleting and re-adding. Did not help.
Thanks!

![image](https://user-images.githubusercontent.com/15839016/145067746-70957ae9-d869-4914-941c-670431f77305.png)

in log:
Debug Received Zigbee message from '0xcc86ecfffef3bcae', type 'attributeReport', cluster 'genBasic', data '{"65506":31,"65508":0,"appVersion":68}' from endpoint 1 with groupID 0
Warning Received message from unsupported device with Zigbee model 'TS0601' and manufacturer name '_TZE200_k6jhsr0q'
Warning Please see: https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html

on map:
![image](https://user-images.githubusercontent.com/15839016/145069497-ed385db4-2126-4a4a-8577-54e7e04005b3.png)

P.S. with TS011F the same story.

**Comment by leha123ra** [comment]
I found [this](https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices/tuya.js) document and grep for 'TS0601_switch_'
it's hard for me.
I will try.

**Reply by prsnls** [reply]
Same for me with 4 gang switch TS0601_switch TZE200_k6jhsr0q.
Paired in Zigbee2mqtt says not supported device, so Homeassistant will not catch it. Need help please

**Comment by RogerioVidigal** [comment]
Same issue here... Anyone found a solution???

**Comment by Koenkk** [comment]
Added the `TZE200_k6jhsr0q` 

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by prsnls** [reply]
Thx so much
